src/pages/nomad/[customDocsLanding].tsx

```diff
@@ -0,0 +1,47 @@
+import { ProductData } from 'types/products'
+import ProductRootDocsPathLanding from 'views/product-root-docs-path-landing'
+import { generateGetStaticProps } from 'views/product-root-docs-path-landing/server'
+// product data
+import nomadData from 'data/nomad.json'
+// page content, which we read in from filesystem
+import fs from 'fs'
+import path from 'path'
+
+export async function getStaticPaths() {
```

Comment:
This `[customDocsLanding].tsx` page file renders all custom docs landing pages for Nomad. Could potentially abstract this approach to other products, maybe combine with the approach in #716 , but in this PR I'm trying to keep the scope relatively tight to Nomad only.